Formula/mujs.rb

```diff
@@ -18,6 +18,10 @@ class Mujs < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d6d7d974cb43e22f2b81342824b0f487d1c3e08af0ebfd3bc37ba77aafb870b6"
   end
 
+  on_macos do
+    patch :DATA
```

Comment:
We don't carry patches indefinitely in Homebrew, so this patch will need to be upstreamed in some form first.

Reply:
However I don't think they will accept patches that targets macOS only

Reply:
Yes, the patch needs to be adjusted to work on both macOS and non-macOS, which is why I said "in some form".

What issues are you having with the shared libraries ending in `.so` instead of `.dylib`?

Reply:
It confuses me and may confuse the linker too.

Anyway, I have submited a [PR](https://github.com/ccxvii/mujs/pull/155/files) but I don't expect it will be merged but just to obey homebrew's rule.